CHANGELOG.md

```diff
@@ -2,24 +2,35 @@
 
 ## Unreleased
 
-## Version 0.4.0 *(2024-10-16)*
+## [0.4.0] *(2024-10-16)*
+
+**Added**
 
  * New: Require JDK 1.8+.
  * New: Run the first specialization when launching from the IDE.
 
 
-## Version 0.3.0 *(2024-10-15)*
+## [0.3.0] *(2024-10-15)*
+
+**Fixed**
 
  * Fix: Don't generate invalid bytecode by attaching a test class constructor to its enclosing file.
 
 
-## Version 0.3.0 *(2024-10-10)*
+## [0.2.0] *(2024-10-10)*
+
+**Added**
 
  * New: Support both class constructor parameters and test function parameters.
 
 
-## Version 0.1.0 *(2024-10-08)*
+## [0.1.0] *(2024-10-08)*
 
 Initial release. We're rebooting the [Burst] project that released 1.0 ten years ago.
 
 [Burst]: https://github.com/square/burst
+[Unreleased]: https://github.com/cashapp/burst/compare/0.4.0...HEAD
+[0.4.0]: https://github.com/cashapp/burst/releases/tag/0.4.0
```

Comment:
Please put the links directly below the headings

Reply:
Updated. GH didn't show the updated state :)